drivers/hmis/lib/form_data/default/assessments/base_exit.json

```diff
@@ -37,7 +64,21 @@
       "fragment": "#disability"
```

Comment:
imo disability should stay as 1 fragment because it has dependencies across the items (disabling condition and "disability table")